samples/CreateSource.java

```diff
@@ -1,22 +1,24 @@
-import com.coveo.pushapiclient.Region;
-import com.coveo.pushapiclient.PlatformUrl;
-import com.coveo.pushapiclient.PlatformUrlBuilder;
-import com.coveo.pushapiclient.Source;
-import com.coveo.pushapiclient.SourceVisibility;
+import com.coveo.platform.Environment;
+import com.coveo.platform.Region;
+import com.coveo.platform.PlatformUrl;
+import com.coveo.platform.PlatformUrlBuilder;
+import com.coveo.source.SourceClient;
+import com.coveo.source.SourceVisibility;
 
 import java.io.IOException;
 import java.net.http.HttpResponse;
 
 public class CreateSource {
     public static void main(String[] args) {
         PlatformUrl platformUrl = new PlatformUrlBuilder()
-                .withRegion(Region.AU) // If your organization is located in a different region than Region.US
+                .withEnvironment(Environment.STAGING)
+                .withRegion(Region.US) // If your organization is located in a different region than Region.US
```

Comment:
Why changing to the default region. The goal of that code snippet is to show user how to switch to another region (other than the default US)

Reply:
mistake :) i was just playing with it to make sure it is working , good catch , i will put it back